src/number_theory/bertrand.lean

```diff
@@ -17,6 +17,26 @@ private def α (n : nat) (pos : 0 < n) (p : nat) (is_prime : nat.prime p) : nat
     exact (multiplicity.finite_iff_dom.1 fin),
   end
 
+def primes_le (n : nat) : finset {m : nat // m ≤ n ∧ nat.prime m} :=
```

Comment:
I like the `\N` unicode symbol for natural numbers. Also, this should be `finset \N := _` where the `_` could very well be `(finset.range (n+1)).filter nat.prime`.

Reply:
Yeah, I'll lint `nat` into `\N` when I actually come to merge this, but `nat` is easier to type and search for than `\N` so that's the style I use for development.